### PR TITLE
Mes 1918 detail test outcome

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
MES 1918 - show serious, dangerous and driving faults summary on test outcome screen.

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [x] PO's approval

fullsizeoutput_d

### CODE REVIEW

File: src/pages/debrief/debrief.html [hunk 1]
```diff
@@ -19,6 +19,72 @@ <h1>unsuccessful</h1>
 
 <ion-content>
 
+  <ion-card id="dangerous-fault" *ngIf="(pageState.dangerousFaults$ | async).length > 0">
+    <ion-card-header>
+      <ion-card-title>
+        <h1 style="display:inline">{{(pageState.dangerousFaults$ | async).length}}</h1> <h4 style="display:inline">Dangerous</h4>
```
Comment (johnmorleybjss) [inline src/pages/debrief/debrief.html]: Could do with getting rid of these inline styles (all severity levels).

Comment (rstepnowski) [inline src/pages/debrief/debrief.html]: Can we get rid of the ```display:inline``` styles and move them to sass? It seems like a recurring theme and probably will be used on the office page as well...

File: src/pages/debrief/debrief.ts [hunk 2]
```diff
@@ -28,6 +48,49 @@ export class DebriefPage extends BasePageComponent {
...
+
+    const { seriousFaults$, dangerousFaults$, drivingFaults$ } = this.pageState;
+
+    const merged$ = merge(
+      seriousFaults$,
+      dangerousFaults$,
+      drivingFaults$,
+    );
+
+    this.subscription = merged$.subscribe();
```
Comment (johnmorleybjss) [inline src/pages/debrief/debrief.ts]: Need to unsubscribe from this at some point.

File: src/pages/debrief/debrief.html (hunk 1 above)
Comment (rossfauldsbjss) [reply src/pages/debrief/debrief.html]: added to scss

File: src/pages/debrief/debrief.ts (hunk 2 above)
Comment (rossfauldsbjss) [reply src/pages/debrief/debrief.ts]: added an unsubscribe.

File: src/pages/debrief/debrief.html (hunk 1 above)
Comment (rossfauldsbjss) [reply src/pages/debrief/debrief.html]: Have aded them to css (sass)

**Pull request merged**